terraform/Makefile

```diff
@@ -19,6 +18,7 @@ ifeq ($(NEED_OSCLOUD),1)
 else
   ENVIRONMENT ?= $(OS_CLOUD)
 endif
+USERNAME=$(shell ( grep '^ssh_username' environments/environment-$(ENVIRONMENT).tfvars || echo ubuntu ) | $(SED) 's@^ssh_username[^=]*= *"*\([^"]*\).*$$@\1@' )
```

Comment:
Thx for providing this patch. IMO it was unintended to not use `environment-.tfvars` ssh_username variable as a source for `$USERNAME` here. 

Would be great to adjust the relevant doc as well. I found only one place: https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/main/doc/make-reference.md#make-ssh

Reply:
ok, good hint i will change that as well